bazel.sh

```diff
@@ -130,5 +129,5 @@ ls /Applications/ | grep "Xcode" | while read -r xcode_path; do
   fi
 
   bazel clean
-  bazel $ACTION $TARGET --xcode_version $xcode_version $extra_args $verbosity_flags $BAZEL_ARGS
+  bazel $ACTION $TARGET --xcode_version $xcode_version $extra_args $verbosity_flags "${POSITIONAL[@]:2}"
```

Comment:
Why did it change from 3 to 2?

Reply:
I'm confused by this as well - seems like the two different expansion styles have different offsets?

Reply:
Ok. Your approach looks reasonable: slicing the original array starting (and including) with the third element.

https://stackoverflow.com/questions/1335815/how-to-slice-an-array-in-bash